apps/api/src/config/index.ts

```diff
@@ -4,14 +4,18 @@ import path from "path";
 import {
   CROSS_CHAIN_TRADING_TYPE,
   CrossChainTradingType,
+
```

Comment:
misc changes that should get caught in CI, but they dont. locally, they _do_ cause an error